scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1060,6 +1062,10 @@ void BPF_STRUCT_OPS(layered_enqueue, struct task_struct *p, u64 enq_flags)
 	if (vtime_before(vtime, layer->vtime_now - layer_slice_ns))
 		vtime = layer->vtime_now - layer_slice_ns;
 
+	if (!tctx->all_cpus_allowed){
```

Comment:
topo being wrong means that all that is needed to fix stalls might be this 4 lines, testing w/ topo disabled to see if stall or nostall w/o the additional lines.